app/src/main/java/sims/michael/gerritslackbot/slack/EventGroupToSlackMessageTransformer.kt

```diff
@@ -148,7 +148,7 @@ class EventGroupToSlackMessageTransformer(
     }
 
     private fun EventGroup<*>.toProjectBranchPrefix(): String {
-        val baseUrl = HttpUrl.parse(events.first().change.url).let { "${it.scheme()}://${it.host()}" }
+        val baseUrl =  events.first().gerritBaseUrl()
```

Comment:
Why replace `okhttp3.HttpUrl#parse` with a regex approach? If it's due to needing the port number, that can still be obtained from `okhttp3.HttpUrl`. I'd prefer to use `HttpUrl` because its parsing is much more comprehensive than the regex. Thoughts?

Reply:
My installation uses http reverse proxy so actual URL in received events was some thing like 

http://server:8976/node1/gerrit/4578

So just getting port, host and scheme from okhttp and then adding change number was not working for me. That's why this approach. But maybe this can be done better as I do not know okhttp.

Any way this solution was not enough as http://server:8976 was working inside corporate network and Gerrit installation had different address outside of it. Theta why additional property in config file.